src/Commands/DocsCommand.ts

```diff
@@ -127,11 +127,14 @@ export class DocsCommand extends CommandBase {
 
                 });
 
-            } else {
+            } else if (result !== null) {
```

Comment:
There are actually three different cases, and one of them shouldn't return anything.
Case 1: language and term exist. Then you return the article like normal.
Case 2: language exists, term doesn't. Then the error dialogue in the else if is returned.
Case 3: language doesn't exist. Then nothing should be returned. That's because other commands like `#list` aren't languages so there shouldn't be an error.

Originally I used just 'else', but because the 'run' command receives ALL commands starting with a #, it would print out the error dialogue too. 'else if' makes sense in this situation cause its basically:
```
if (term exists) {
    return normal
} else if (term doesn't exist, but result does) {
    return error
} else {
    return nothing
} 
```
But the `return nothing` is left out.
If the future the best way to refactor this imo is to prevent run from see non-language command keywords.